mosip/longhorn/delete.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Uninstalls longhorn
+## Usage: ./delete.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=longhorn-system
+
+while true; do
+    read -p "Are you sure you want to delete apitestrig helm charts?(Y/n) " yn
+    if [ $yn = "Y" ]
+      then
+        echo Removing longhorn
```

Comment:
Add a warning saying the services using longhorn as storage class will loose its data  . do you still want to continue.
and also add a comment saying backup the data and uninstll the required services then go for uninstallation.